roles/foreman_installer/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Check that necessary variables are defined
+  assert:
+    that:
+      - foreman_installer_scenario is defined
+
+- name: Check if initial install has happened
+  find:
+    paths: /etc/foreman-installer/scenarios.d
+    patterns: .installed
```

Comment:
this tests for the presence of `/etc/foreman-installer/scenarios.d/.installed`, right?

I'd have used the `stat` module and checked `result.exists`